src/sentry/static/sentry/app/views/organizationDashboard/discoveryQuery.jsx

```diff
@@ -0,0 +1,133 @@
+import PropTypes from 'prop-types';
```

Comment:
nit: filename should be `discoverQuery` vs `discoveryQuery`

Reply:
thanks, fixed https://github.com/getsentry/sentry/pull/11416/commits/73039827f867a437c61918a36c7d9190c6cd0961